src/worker/sandbox/cgroups/pool.go

```diff
@@ -23,20 +22,52 @@ type CgroupPool struct {
 	recycled chan *CgroupImpl
 	quit     chan chan bool
 	nextID   int
+	logger   slog.Logger
 }
 
 func NewCgroupPool(name string) (*CgroupPool, error) {
+
+	level := new(slog.LevelVar)
+	if common.Conf.Trace.Cgroups_level == "INFO" {
+		level.Set(slog.LevelInfo)
+	} else if common.Conf.Trace.Cgroups_level == "WARN" {
+		level.Set(slog.LevelWarn)
+	} else if common.Conf.Trace.Cgroups_level == "ERROR" {
+		level.Set(slog.LevelError)
+	}
+	// Default logger right now can't have the level be changed (haven't found the API for that, probably have to implement one from the interface)
+	logger := slog.Default()
+
+	// Replacing default logger based on config
+	if common.Conf.Trace.Format == "text" {
+		logFilePath := path.Join(common.Conf.Trace.Log_file_dir, "log.txt")
```

Comment:
Shouldn't this be created at a much more general level?  I imagine this log file is shared across many different subsystems.